### DISCUSSION TITLE
[BDH-14] Default Ruby version is still 2.7?

### DISCUSSION THREAD

**Discussion by Dvalon** [vercel/community#5599] [Help] [answered]
### Summary

I tried to deploy a Jekyll project and state the ruby is 3.2.1 in the Gemfile. But it pops up "Your Ruby version is 2.7.0, but your Gemfile specified ~> 3.2.1" when build. I know all ruby version below 3.0 were blocked last year so don't know why it said my environment is 2.7 still.

### Example
<img width="875" alt="Screenshot 2024-01-25 at 11 33 54" src="https://github.com/vercel/community/assets/67376370/1675a064-ccc1-4cce-bfbe-7c89853ea141">

_No response_

### Steps to Reproduce

Using the project: https://github.com/cotes2020/jekyll-theme-chirpy

**Comment by swarnava** [comment]
Hi,

Let us check this and get back to you!

**Reply by Dvalon** [reply]
Hi. Do you have any updates?

**Reply by swarnava** [reply]
Hi, unfortunately I don't. We are still investigating this internally.

**Reply by swarnava** [reply]
This will be fixed on next version of CLI with https://github.com/vercel/vercel/pull/11104

**Comment by swarnava** [comment, marked as answer]
This issue is fixed now!
